packages/firestore/src/local/local_documents_view.ts

```diff
@@ -84,28 +106,40 @@ export class LocalDocumentsView {
     transaction: PersistenceTransaction,
     keys: DocumentKeySet
   ): PersistencePromise<MaybeDocumentMap> {
+    return this.remoteDocumentCache
+      .getEntries(transaction, keys)
+      .next(docs => this.getLocalViewOfDocuments(transaction, docs));
+  }
+
+  /**
+   * Similar to `getDocuments`, but creates the local view from the given
+   * `baseDocs` without retrieving documents from the local store.
+   */
+  getLocalViewOfDocuments(
+    transaction: PersistenceTransaction,
+    baseDocs: NullableMaybeDocumentMap
+  ): PersistencePromise<MaybeDocumentMap> {
+    let allKeys = documentKeySet();
```

Comment:
Can you try using `AnyJs` instead of `any`?  

any is dangerous in that it basically opts out of typechecking.  So if you type the values as `any`, TypeScript will let you do anything with the values (call nonexistent methods, etc.) which is the opposite of what we want (we don't want to be able to do anything with the values).

`AnyJs` is a type we introduced essentially meant to be a supertype of any JS type (similar to `Object` in java). So TypeScript won't let you do anything with it without doing typeof / instanceof checks to narrow it down.

TypeScript 3.x actually introduced `unknown` which is similar to AnyJs and I'm hoping we can migrate to it soon.

Reply:
Sorry, missed this comment. Done now.